mautic/mautic
# New Campaign settings: Disable failed notifications for do not contact

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) |  https://github.com/mautic/mautic/issues/4494,  partially https://github.com/mautic/mautic/issues/5494
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Add new option "Disable failed notifications for do not contact " to Campaign settings 

![image](https://user-images.githubusercontent.com/462477/36570219-4f5bbb0c-1831-11e8-91d5-78335121ccec.png)

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create campaign, add contact marked as DNC and send email action
2.  Run campaign
3. See If failed notification will add in Mautic notification area

#### Steps to tests PR
0. Apply PR and clearch cache and go to Configurations > Campaign settings and enable new option "Disable failed notifications for do not contact"
1.  Create campaign, add contact marked as DNC and add send email action 
2.  Run campaign
3. See If contact wasn't  add in Mautic notification area
